salepoint/app/Partner.php

```diff
@@ -66,10 +81,25 @@ public static function filterAndPaginateCustomer($name)
             ->orderBy('partners.name','ASC')
             ->paginate();
 
+    }*/
+    public static function filterAndPaginateCustomer($name)
+    {
+        return \DB::table('selectPartners')
+            ->where('customer','=',1,'and','name','like','%'.$name.'%')
```

Comment:
Aqui esta linea no quiero decir que esta mal... pero hay que seguir el estandar de laravel

si quieres agregar un and es agregando un where() .... si quieres agregar un or es un orwhere()

Ejemplo:
AND
->where(condicion1)
->where(condicion2)

OR
->where(condicion1)
->orwhere(condicion2)